api/src/main/java/org/openmrs/module/isanteplusreports/healthqual/util/RegisterAllHealthQualReports.java

```diff
@@ -276,6 +279,11 @@ private static void healthQualPediatricNegativePcrTest() {
             HealthQualReportsConstants.HEALTH_QUAL_PEDIATRIC_13_INDICATOR_UUID);
     }
 
+    private static void healthQualEarlyInfantDiagnosis() {
+        registerHealthEqualReportWithStartAndEndDateParams(PEDIATRIC_14_INDICATOR_SQL, PEDIATRIC_14_INDICATOR_MESSAGE,
+            HealthQualReportsConstants.HEALTH_QUAL_PEDIATRIC_14_INDICATOR_UUID);
+    }
+    
     private static void registerHealthEqualReportWithStartAndEndDateParams(String sql, String messageProperties, String uuid) {
         SqlDataSetDefinition sqlData = sqlDataSetDefinitionWithResourcePath(sql, messageProperties, messageProperties, HEALTH_QUAL_REPORTS_RESOURCE_PATH);
```

Comment:
You are passing messageProperties parameter twice?

Reply:
One parameter is meant to capture the name while the other is meant to capture the description of the sqldataset definition instance.